### PR TITLE
Removed failed init for devices that dont support UpsideDown paramete…

### PR DESCRIPTION
**Pull request by fehlfarbe**
Removed failed init for devices that dont support UpsideDown parameter (see: https://github.com/SICKAG/sick_ldmrs_laser/issues/1)

### CODE REVIEW

**Comment by mintar** [comment]
Thanks for your PR! Did your test this? On which device? @DottorZini 's LD-MRS400001, perhaps?

**Comment by fehlfarbe** [comment]
I tested this on LD-MRS400102 HD

**Comment by mintar** [comment]
Ok, good to know! Is this the only change required? In other words, is it working now?

**Comment by fehlfarbe** [comment]
Yes, I got the same error messages as described in SICKAG/sick_ldmrs_laser#1 with my LD-MRS400102 HD. 
With the changes it prints the error message `ERROR: LuxBase::readUpsideDown: LD-MRS ERROR: Failed to read UpsideDown flag, aborting.` but it doesn't abort the driver initialization and the device is working.

File: src/devices/LuxBase.cpp [hunk 1]
```diff
@@ -215,12 +216,14 @@ bool LuxBase::initTcp(Tcp::DisconnectFunction function, void* obj)	// , bool beV
...
 	}
+/*
+// some devices dont have the UpsideDown flag so just ignore this error
 	else
 	{
 		printError("LuxBase::initTcp(): ERROR: Failed to read UpsideDown flag, aborting.");
 		return false;
 	}
-
+*/
```
Comment (jspricke) [inline src/devices/LuxBase.cpp]: Please don't commit commented code. Better print an informative message and return true.

File: src/devices/LuxBase.cpp [hunk 2]
```diff
@@ -200,6 +200,7 @@ bool LuxBase::initTcp(Tcp::DisconnectFunction function, void* obj)	// , bool beV
 	// Read the UpsideDown mode.
 	// We need this parameter later for the calculation of cartesian scan point coordinates.
 	//
+
```
Comment (jspricke) [inline src/devices/LuxBase.cpp]: please don't add unrelated changes

**Comment by jspricke** [comment]
@DottorZini can you test this before we merge?

**Comment by DottorZini** [comment]
I'll test it as soon as possible. Anyway I confirm that these changes are identical to what I previously did to make it work.

**Comment by fehlfarbe** [comment]
I tested with an IBEO LUX Standard that should be similar to LD-MRS400001/LD-MRS400001S01 and it works without problems with pointcloud, object tracking and so on.

**Comment by mintar** [comment]
@fehlfarbe : You said before that you tested this on an LD-MRS400102 HD. Which is it? Thanks! :)

**Comment by fehlfarbe** [comment]
@mintar we have both devices, LD-MRS400102 HD and IBEO LUX Standard, in our laboratory and it works with both.

**Comment by mintar** [comment]
Merged via 7d6c83a. Thanks a lot!
